<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditConnectionDialog</class>
 <widget class="QDialog" name="AddEditConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conexão PostGIS</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Nome da Conexão:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblHost">
       <property name="text">
        <string>Host:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtHost"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblPort">
       <property name="text">
        <string>Porta:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtPort">
       <property name="text">
        <string>5432</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblDatabase">
       <property name="text">
        <string>Banco de Dados:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtDatabase"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblUsername">
       <property name="text">
        <string>Usuário:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="txtUsername"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblPassword">
       <property name="text">
        <string>Senha:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="txtPassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblService">
       <property name="text">
        <string>Serviço (opcional):</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="txtService"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lblSslMode">
       <property name="text">
        <string>Modo SSL:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="cmbSslMode">
       <item>
        <property name="text">
         <string>disable</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>allow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prefer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>require</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnTest">
       <property name="text">
        <string>Testar Conexão</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

